### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by mkillianey**

### CODE REVIEW

**Comment by mkillianey** [comment]
@HenrikJoreteg -- I know that you indicated the MIT license, but I'm using this package indirectly through other packages, and I just wanted to confirm that this is the text you'd like to use to acknowledge your work.

**Comment by HenrikJoreteg** [comment]
I typically link to: http://mit.joreteg.com/ would adding that to the readme suffice for what you need?

**Comment by HenrikJoreteg** [comment]
I've linked to it from the readme and re-published as `1.0.2` thanks!

**Comment by mkillianey** [comment]
@HenrikJoreteg -- This is such a trivial library that I'm happy to pull the text off the link.  In general, your code would be easier to use if there was a LICENSE file, for at least three reasons:

(1)  If you have a file named LICENSE, it gets automatically included in your package when you publish, so it's easier for me (or the next person who uses this) to find exactly the right text to include...I can automate the process using something like https://www.npmjs.com/package/license-checker with zero effort.

(2)  Without the LICENSE file, the version of this package published on npm seems like it violates your own license, since MIT says that the copyright notice shall be included in all copies of the software, and I don't see the copyright notice in https://registry.npmjs.org/tryit/-/tryit-1.0.2.tgz

(3)  I get if you're trying to simplify by listing one copy of MIT license for everything you open-source, but in this case, it makes it more confusing to understand how it can be used.  For example, the copyright in the license for this package should be (c)2013-2015, not (c)2015, as is on your website.  Are you declining copyright for the work you did in 2013 and 2014, and if so, could someone else claim ownership of that copyright and invalidate yours?  Or, what if MIT revises it's license tomorrow...will you change the contents of that page to be the new license?  If so, then it's confusing to me as a user whether it would still be ok to use your old stuff under the _old_ contents of that page, or if I'm obliged to use the _new_ contents of that page.  Or what if you stop maintaining your website and I can't get to the license file anymore...do I have to stop using your stuff?  Unless there's some reason to want to introduce these kinds of complexity, it's just easier for me to understand your terms of use if there's a LICENSE file that ships with the package.

In short, for my selfish purposes, what you've provided is good enough for me.  For the greater good, though, I'll just point out that if you're willing to do so, including a LICENSE file does make life simpler/unambiguous for people who are just trying to Do the Right Thing by you, legally.  (I see that you're a generous open-sourcer, and many of your repos have license files, but some don't, so maybe consider this a long-winded request to consider including license files in your source repos, rather than links.)
